<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CatchmentsDialogBase</class>
 <widget class="QDockWidget" name="CatchmentsDialogBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>543</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>543</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Catchments</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QVBoxLayout" name="apiLayout">
      <item>
       <widget class="QLabel" name="providerLabel">
        <property name="text">
         <string>API Provider</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="providersComboBox"/>
      </item>
      <item>
       <widget class="QLabel" name="keyLabel">
        <property name="text">
         <string>Key</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLineEdit" name="keyLineEdit"/>
        </item>
        <item>
         <widget class="QLabel" name="getKeyLabel">
          <property name="text">
           <string/>
          </property>
          <property name="textFormat">
           <enum>Qt::RichText</enum>
          </property>
          <property name="openExternalLinks">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="layersLayout">
      <item>
       <widget class="QLabel" name="layersLabel">
        <property name="text">
         <string>Select point layer</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="layersFieldsLayout">
      <item>
       <widget class="QLabel" name="fieldIdLabel">
        <property name="text">
         <string>Select layer ID field</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QgsFieldComboBox" name="fieldsComboBox">
        <property name="allowEmptyFieldName">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QCheckBox" name="selectCheckBox">
      <property name="text">
       <string>Only selected</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="pointsLabel">
      <property name="text">
       <string>Number of points: 0</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="Line" name="line_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="modeLabel">
      <property name="text">
       <string>Mode</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="modesComboBox"/>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Value and unit</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QSpinBox" name="valueSpinBox"/>
      </item>
      <item>
       <widget class="QComboBox" name="unitsComboBox"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="avoidLabel">
      <property name="text">
       <string>Parameters</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="trafficCheckBox">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Include traffic</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="pointsRouteStartCheckBox">
      <property name="text">
       <string>Points as route start</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="getCatchments">
      <property name="text">
       <string>Get catchments</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsFieldComboBox</class>
   <extends>QComboBox</extends>
   <header>qgsfieldcombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
